conan/qasm/conandata.yml

```diff
@@ -1,5 +1,5 @@
 sources:
-  hash: "d59248cbb4cf8840c9720462f2569907b9506323"
+  hash: "d42bca2c2b7a8e2543462cceea178240709b6fa2"
```

Comment:
Would you mind updating this to https://github.com/openqasm/qe-qasm/commit/2a7cb619d28fc45b9b91460468f42f47e905573b?